src/MOI_wrapper.jl

```diff
@@ -330,23 +389,53 @@ function MOI.get(
         ci_dual = primal_dual_map.constrained_var_dual[ci_primal]
         return _get_at_index(
             optimizer,
-            MOI.ConstraintDual(),
+            _dual_attribute(attr),
             ci_primal,
             ci_dual,
             idx,
         )
     else
         return -MOI.get(
             optimizer.dual_problem.dual_model,
-            MOI.ConstraintDual(),
+            _dual_attribute(attr),
             get_ci_dual_problem(optimizer, vi),
         )
     end
 end
 
+function MOI.supports(
+    ::DualOptimizer,
+    ::Union{MOI.ConstraintDualStart,MOI.ConstraintPrimalStart},
+    ::Type{<:MOI.ConstraintIndex},
+)
+    return true
```

Comment:
Needs to check the inner optimizer supports `VariablePrimalStart`. I don't know if `ConstraintPrimalStart` makes sense to support?